### PR TITLE
Fix: add generate JDBC connection workflowDefintion

### PR DESCRIPTION
**Pull request by zzxwill**
Added a WorkflowDefinition

Signed-off-by: Zheng Xi Zhou <zzxwill@gmail.com>

### Description of your changes

Fixes #

I have:

- [x] Read and followed KubeVela's [contribution process](https://github.com/oam-dev/kubevela/blob/master/contribute/create-pull-request.md).
- [x] [Related Docs](https://github.com/oam-dev/kubevela.io) updated properly. In a new feature or configuration option, an update to the documentation is necessary. 
- [ ] Run `make reviewable` to ensure this PR is ready for review.
- [ ] Added `backport release-x.y` labels to auto-backport this PR if necessary.

### CODE REVIEW

**Comment by zzxwill** [comment]
The documentation for this workflowstep is https://github.com/oam-dev/kubevela.io/pull/531

**Review by wonderflow — APPROVED** [review]
good job, please also add docs in  kubevela.io

File: vela-templates/definitions/internal/workflowstep/generate-jdbc-connection.cue [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+		}
+	}
+	dbHost:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_HOST"])}
+	dbPort:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_PORT"])}
+	dbName:   op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_NAME"])}
+	username: op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_USER"])}
+	password: op.#ConvertString & {bt: base64.Decode(null, output.value.data["DB_PASSWORD"])}
+
+	env: [
+		{name: "url", value:      "jdbc://" + dbHost.str + ":" + dbPort.str + "/" + dbName.str + "?characterEncoding=utf8&useSSL=false"},
```
Comment (barnettZQG) [inline vela-templates/definitions/internal/workflowstep/generate-jdbc-connection.cue:34]: advised to make it a configurable parameter, `characterEncoding=utf8&useSSL=false` This is not suitable for all.

others: Can env be injected directly?

Comment (zzxwill) [reply vela-templates/definitions/internal/workflowstep/generate-jdbc-connection.cue:34]: Yes, you are right. Let's make it simple and clear at the very beginning. And wait and hear feedback from end-users. I will expose them to users after that. Thanks.

**Pull request merged**